src/hydrate/values/getHydrationDefaults.ts

```diff
@@ -5,20 +5,20 @@ import { readAuthorIfExists } from "./readAuthorIfExists.js";
 import { readEmailIfExists } from "./readEmailIfExists.js";
 import { readFundingIfExists } from "./readFundingIfExists.js";
 import { readOwnerFromGitRemote } from "./readOwnerFromGitRemote.js";
+import { readTitleFromReadme } from "./readTitleFromReadme.js";
 
 export async function getHydrationDefaults() {
-	const existingReadme = await readFileSafe("./README.md", "");
 	const existingPackage = JSON.parse(
 		await readFileSafe("./package.json", "{}")
 	) as PartialPackageData;
 
 	return {
-		author: () => readAuthorIfExists(existingPackage),
-		email: () => readEmailIfExists(existingPackage),
-		funding: readFundingIfExists,
-		owner: readOwnerFromGitRemote,
+		author: await readAuthorIfExists(existingPackage),
+		email: await readEmailIfExists(existingPackage),
+		funding: await readFundingIfExists(),
+		owner: await readOwnerFromGitRemote(),
 		releases: true,
-		title: existingReadme.match(/^(?:# |<h1\s+align="center">)(\S+)/)?.[1],
+		title: await readTitleFromReadme(),
```

Comment:
[Request] Ah, the reason why these are `() =>` to begin with is so that they don't have to be evaluated if not needed.

```suggestion
		title: readTitleFromReadme,
```